### ISSUE TITLE
Allow control over buttons

### ISSUE THREAD

**Issue by natew** [mobxjs/mobx-react-devtools#21]
Rather than giving us a pre-styled output, could we have access to just the events for each button?

A step above that would be to provide a way to get each icon individually.

Final step above that would be to allow not having position fixed on the entire devtools.

Use case is that we don't want it fixed in our app but in a dropdown that we show, so any of the solutions would help.

**Comment by andykog** [comment]
Thats absolutely possible.

**Comment by andykog** [comment]
@natew, with my latest changes (not released yet) you can do it like this:

``` js
import DevTool, { GraphControl, LogControl, UpdatesControl } from 'mobx-react-devtools';

const MyNiceButton =
  ({ active, onToggle }) => <button onClick={onToggle}>{active ? 'on' : 'off'}</button>;

class MyApp extends React.Component {
  render() {
    return (
      <div className="my-custom-devtools-panel-design">
        <DevTool noPanel />
        <GraphControl>
          <MyNiceButton />
        </GraphControl>
        <LogControl>
          <MyNiceButton />
        </LogControl>
        <UpdatesControl>
          <MyNiceButton />
        </UpdatesControl>
      </div>
    );
  }
}
```

Any `*Control` must have only one child that takes props: `active` (bool), `onToggle` (func) 

Is that what you needed?

**Comment by natew** [comment]
That looks awesome, thanks

**Comment by andykog** [comment]
Oh, and you will still need to include `<Devtool noPanel />` somewhere to display updates and modals

**Comment by andykog** [comment]
@mweststrate, could you please review and release this as well as #23

**Comment by mweststrate** [comment]
@andykog reviewed and released as 4.2.1. Would you mind updating the README with these new changes? Thanks!

**Comment by andykog** [comment]
@mweststrate, done https://github.com/mobxjs/mobx-react-devtools/commit/bdb7457ede10d0be77069a89662db7ebca05c844
